sdk/communication/Azure.Communication.Chat/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0-beta.5 (2021-03-09)
 Updated `Azure.Communication.Chat` version.
```

Comment:
This line can be removed since it was added when there was no content

Reply:
Thanks @ankitarorabit, I removed the line in the other open PR: https://github.com/Azure/azure-sdk-for-net/pull/19387/files#diff-41854cbbfb20e38314beeaf2d0d51b6b8f664b2e8200d9732ffa7b4bc834fa3f